submissions/masturohAl-Ameen/html-forms/signUp.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Contact Us</title>
+    <link rel="stylesheet" type="text/css" href="../css/signUp.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap" rel="stylesheet">
+</head>
+<body>
+    <nav>
+        <a href="productpage.html"><h1 id="logo"><b>INNER GLAMOUR</b></h1></a>
+        <ul id="menu">
+            <li><a href="../HTML/productpage.html">Product Page</a></li>
+            <li ><a href="contactUs.html">Contact Us</a></li>
+            <li><a href="#">Sign-up</a></li>
+        </ul>
+    </nav>
+
+    <section class="customers-form">
+        <div class="register">
+            <h2>Register your Account</h2>
+            <P>Registering an account gives you the opportunity to get information and access to promos and events available to only registered customers</P>
```

Comment:
It should be a p tag and not P. Also, this line of code looks long. Kindly use the enter key to break it to the next line.